ghost/core/core/server/api/endpoints/utils/serializers/output/mappers/emails.js

```diff
@@ -19,7 +18,7 @@ module.exports = (model, frame) => {
         }
     }
 
-    if (!labs.isSet('emailErrors') && !!(config.get('bulkEmail') && config.get('bulkEmail').mailgun)) {
+    if (config.get('bulkEmail') && config.get('bulkEmail').mailgun) {
```

Comment:
@SimonBackx Correct me if I'm wrong but removing the flag would actually involve removing this whole block because we don't want to remap the status

Reply:
That is right, the whole block needs to be removed. Currently `labs.isSet('emailErrors')` always returns true because it is a GA flag, so this block never executes. What this block was doing, was making sure that any API users (such as admin) never received a status of 'failed', so they would never show an error message when an email failed to send.

Reply:
Good catch @daniellockyer thanks for the heads up

Nice one for the explanation @SimonBackx 

I've updated this to remove the whole block now 👍